index.html

```diff
@@ -1,14 +1,30 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
 <head>
-    <meta charset="UTF-8">
-    <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-    <meta http-equiv="X-UA-Compatible" content="ie=edge">
-    <title>Homework 4</title>
-    <link rel="stylesheet" href="./style.css">
+	<meta charset="UTF-8">
+	<meta http-equiv="X-UA-Compatible" content="IE=edge">
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<link rel="stylesheet" href="./styles/all.css">
+	<link rel="stylesheet" href="./styles/style.css">
+	<title>Document</title>
 </head>
 <body>
-
+	
+		<div class="frame">
+			<div class="profile"> <img src="./82ab3533ee71daf256f23c1ccf20ad6f--avatar-maker.jpg" alt=""> <p>CIELO TENNERT</p> <br> <p>Web developer</p>
+				<div class="button">
+					<button class="button1" type="button"><span> <i class="fas fa-heart"></i> Like</span></button> <br>
+					<button class="button2" type="button"><span><i class="fas fa-thumbs-up"></i> Follow</span></button> <br>
+					<button class="button3" type="button"><span> <i class="fas fa-envelope"></i> Message</span></button> 
+				</div>
+			</div>
+			<div class="statistics">
+				<div class="likes"><p>2021 Likes</p></div>
```

Comment:
you need to add Likes to `span`
the same with Posts and Followers

and on hover, this text on span should change colour, only text, without numbers